src/azure-cli/azure/cli/command_modules/sql/_params.py

```diff
@@ -261,10 +261,12 @@ def get_location_type_with_default_from_resource_group(cli_ctx):
 
 aad_admin_login_param_type = CLIArgumentType(
     options_list=['--display-name', '-u'],
+    required=True,
```

Comment:
This will make display_name as required for:
- `az sql server ad-admin create`
- `az sql server ad-admin update`
- `az sql mi ad-admin create`
- `az sql mi ad-admin update`

Are you sure with all the commands change?

Reply:
Yes. These commands don't work without both and this seems to be the case for the underlying REST API as well (The parameter description mentions display name, which maps to `login` in the REST API)

- https://docs.microsoft.com/en-us/rest/api/sql/2021-11-01-preview/managed-instance-administrators/create-or-update
- https://docs.microsoft.com/en-us/rest/api/sql/2021-11-01-preview/server-azure-ad-administrators/create-or-update